src/include/kdb.h.in

```diff
@@ -107,9 +100,9 @@ namespace ckdb {
 extern "C" {
 #endif
 
-typedef struct _KDB	KDB;
-typedef struct _Key	Key;
-typedef struct _KeySet	KeySet;
+typedef struct _KDB	ElektraKDB;
```

Comment:
1. How would you redefine this, without rewriting the entire header file? As soon as you include the header there will be a conflict, if there is already a `KDB`. This will cause a compile error.
2. I think it is easier for us and for users, if we just consistently prefix _everything_ with some form of `Elektra`. Then you know this belongs to Elektra when reading code, we avoid all sorts of conflicts and we don't need to think whether or not to use the prefix.